fastlane/bin/fastlane

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env ruby
 $LOAD_PATH.push File.expand_path("../../lib", __FILE__)
 
+unless ENV['BUNDLE_BIN_PATH']
+  has_gemfile = File.exist?('Gemfile')
+  File.open('Gemfile', 'w') { |file| file.write("gem 'fastlane'") } unless has_gemfile
```

Comment:
Alternatively fastlane could delete temporary Gemfiles itself once it is launched, this way the temporary file is there only during dependency resolving